src/io/record-play-movie.cpp

```diff
@@ -100,45 +101,65 @@ static long get_current_time(void)
 #endif
 }
 
-/* リングバッファ構造体に buf の内容を加える */
-static errr insert_ringbuf(char *buf)
+/* header の内容と、オプションで payload の内容をリング バッファ構造に追加します */
```

Comment:
This comment seems to be changed as Doxygen.
```
/*!
 * @brief リングバッファにヘッダとペイロードを追加する
 * @param header ヘッダ
 * @param payload ペイロード (オプション)
 * @return エラーコード
 */
```

Reply:
The latest forced push adopts your suggested Doxygen comment for insert_ringbuf().